challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
 <!-- Answer Starts Here -->
-
+absolute, because it starts with '/'
 <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
 <!-- Answer Starts Here -->
-
+since we want it remain hosted on some external source we'd rather use the absolute path to refer to it
```

Comment:
You're correct that we use an absolute path, but your explanation is a little off. The reason why we use an absolute path is that we use a URL to refer to something that's hosted on the web, and a URL is an absolute path.